tools/update-translation-files.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+#    Copyright © 2016 Zetok Zalbavar <zetok@openmailbox.org>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Script for updating translation files.  Should be ran after
+# translatable strings are modified.
+#
+# Needed, since weblate cannot into doing that before translation.
```

Comment:
`Needed, since weblate cannot do that before translation.` is better english I think.